app/database/schema/packages.schema.ts

```
@@ -1,5 +1,7 @@
 import { relations } from "drizzle-orm";
 import { boolean, integer, pgTable, serial, text, timestamp } from "drizzle-orm/pg-core";
+import { createInsertSchema } from "drizzle-zod";
```

ESLint

[error] Unable to resolve path to module 'drizzle-zod'.

(import-x/no-unresolved)